native-sdk/src/access_rules/access_rules.rs

```diff
@@ -0,0 +1,32 @@
+use radix_engine_interface::api::node_modules::auth::{
+    AccessRulesCreateInput, ACCESS_RULES_BLUEPRINT, ACCESS_RULES_CREATE_IDENT,
+};
+use radix_engine_interface::api::ClientApi;
+use radix_engine_interface::blueprints::resource::AccessRules;
+use radix_engine_interface::constants::ACCESS_RULES_PACKAGE;
+use radix_engine_interface::data::model::Own;
+use radix_engine_interface::data::{scrypto_decode, scrypto_encode, ScryptoDecode};
+use sbor::rust::fmt::Debug;
+
+pub struct AccessRulesObject;
+
+impl AccessRulesObject {
```

Comment:
Rename to `AccessRules`?

Reply:
Will save the renaming to a future PR since `AccessRules` conflicts with the `AccessRules` rust object used in scrypto.